client/js/widgets/label.js

```diff
@@ -1,5 +1,37 @@
 import { Widget } from './widget.js';
 
+export async function unsetHTML(w) {
+  document.getElementById(w.id).innerHTML = null;
```

Comment:
`document.getElementById(w.id)` should be the same as `w.domElement`.

Reply:
Updated